src/BloomExe/Collection/BookCollection.cs

```diff
@@ -189,7 +187,34 @@ protected Color CoverColor
 			set { throw new NotImplementedException(); }
 		}
 
-		public static string DownloadedBooksCollectionNameInEnglish = "Books From BloomLibrary.org";
+		public const string DownloadedBooksCollectionNameInEnglish = "Books From BloomLibrary.org";
+		private const string InstalledTemplatesCollectionNameInEnglish = "Templates";
+
+		public bool IsDownloaded { get { return Name == DownloadedBooksCollectionNameInEnglish; } }
```

Comment:
The collection isn't downloaded...it's the collection reserved to contain things that are downloaded.